tests/network/flat_overlay/test_multi_network_policy.py

```diff
@@ -6,8 +6,6 @@
 from utilities.network import assert_ping_successful
 
 pytestmark = pytest.mark.usefixtures(
-    "skip_when_one_node",
-    "skip_if_sno_cluster",
```

Comment:
We still need these tests to **not** be executed on SNO/single-worker clusters, so shouldn't they be marked with a relevant marker that will ensure that?

Reply:
on sno, the only tests that runs are the ones with sno markers. This file does not have sno marker, right? Then these tests won't be executed against SNO already. No skip fixture is needed.

Reply:
Ack